src/components/ContactInfo/ContactInfo.vue

```diff
@@ -2,9 +2,9 @@
   <div id="contact-info">
     <!-- Display Contact Info -->
     <v-layout row v-if="!isEditing" class="mx-0">
-      <v-flex xs3>
+      <v-flex xs3 class="pr-10">
```

Comment:
Why add right padding?

For me, right padding is the same as bottom padding, and I try to avoid it.

Also, would it be quick to change this over to using rows and columns (since layout and flex are deprecated)? This might also solve your layout issues.

Reply:
I didn't want to expand the size of this column but wanted to limit the number of words on the first row, so to preserve the alignment, this breaks the text after two words. If there is an alternative to limiting the words in the row to 2 words i'm all for it, wasn't sure what else would work here. 

The `<br>` tag won't work for all iterations, for example the design calls for some words in the template on the first line and in other designs the second line etc. 

I don't believe updating the Vue syntax would solve this, i would prefer to defer this with the scope of work on my plate atm.

Reply:
I could try a character limit in the column vs padding, will try

Reply:
I was wrong, updating the syntax has helped. Thank you 😁